pkg/vendir/directory/directory_test.go

```diff
@@ -0,0 +1,60 @@
+package directory
+
+import "testing"
+
+func Test_handleLazySync(t *testing.T) {
```

Comment:
I am not a fan of testing not exported functions. I understand here that this code would be more complex than testing from the outside. Is there any refactoring that can make this easier?

Reply:
No worries, let's remove it and use the e2e test instead